### PR TITLE
Address bundle-only for data channels. Fixes #505

### LINKED ISSUE #505: Data create-offer section needs to mention bundle-only
Maybe in create-answer as well

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2195,13 +2195,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section
           4.1.</t>
 
-          <t>Within the data m= section, the "a=mid", "a=ice-ufrag",
+          <t>Within the data m= section, an "a=mid" line MUST be
+          generated and included as described above,
+          along with an "a=fmtp:webrtc-datachannel" line and an
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: > along with an "a=sctp-port" line referencing the SCTP port number, as defined in
<xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 5.1, and, if appropriate, an "a=max-message-size" line, as defined in <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 6.1.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2195,13 +2195,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          "a=sctp-port" line referencing the SCTP port number as defined in
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          If the m= section either has a unique address or is associated
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Make a new t section here, and start with "As discussed above, the following attributes are included only if ..."

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -2764,14 +2767,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+          <t>Within the data m= section, an "a=mid" line MUST be
+          generated and included as described above,
           along with an "a=fmtp:webrtc-datachannel" line and an
-          "a=sctp-port" line referencing the SCTP port number as
-          defined in
-          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section
-          4.1.</t>
+          "a=sctp-port" line referencing the SCTP port number as defined in
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          If the m= section either has a unique address or is associated
```
Comment (taylor-b) [inline draft-ietf-rtcweb-jsep.xml]: For the answer, the terminology used above is "not bundled into another m= section", which sounds clearer to me.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -2764,14 +2767,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           "fmt" fields MUST be set to exactly match the fields in the
           offer.</t>
 
-          <t>Within the data m= section, the "a=mid", "a=ice-ufrag",
-          "a=ice-pwd", "a=candidate", "a=fingerprint", "a=dtls-id",
-          and "a=setup" lines MUST be included under the conditions described above,
+          <t>Within the data m= section, an "a=mid" line MUST be
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: See above

**Comment by ekr** [comment]
@juberti: revised

**Pull request merged**